packages/babel-types/test/asserts.js

```diff
@@ -17,18 +16,9 @@ describe("asserts", () => {
       const nodeType = k.replace("assert", "");
 
       it(nodeType, () => {
-        assert.throws(
-          () => {
-            t[k]({ type: "FlavorTownDeclaration" }, {});
-          },
-          err => {
-            return (
-              err instanceof Error &&
-              err.message ===
-                `Expected type "${nodeType}" with option {}, but instead got "FlavorTownDeclaration".`
-            );
-          },
-        );
+        expect(() => {
+          t[k]({ type: "FlavorTownDeclaration" }, {});
+        }).toThrow();
```

Comment:
Can we test the error message?

Reply:
Yes; you can give a constructor to get an `instanceof` check, a regexp to test against the message, or a string to `===` the message (without name).